### ISSUE TITLE
GraphQL::Pagination::Connections removed access to parent on Edge types.

### ISSUE THREAD

**Issue by Mihail-K** [rmosolgo/graphql-ruby#2960]
**Describe the bug**

Using `GraphQL::Pagination::Connections` removes access to `object.parent` on custom Edge types (`GraphQL::Types::Relay::BaseEdge`).

Since Edges are intended to model the relationship between a Node and its parent type, losing access to the parent renders custom Edges largely unusable.

**Versions**

`graphql` version: `1.10.10`
`rails` (or other framework): `6.0.3.1`
`graphq-batch` version: `0.4.3`

**GraphQL schema**

Condensed for brevity:
```ruby
module Types
  class IngredientType < Types::BaseObject
    field :id, ID, null: false
    field :name, String, null: false
    field :description, String, null: true
  end

  class RecipeIngredientEdgeType < Types::BaseEdge
    node_type Types::IngredientType

    field :amount, Float, null: false
    field :unit, String, null: false

    def amount
      recipe_ingredient.then(&:quantity_amount)
    end

    def unit
      recipe_ingredient.then(&:quantity_unit)
    end

  private

    def recipe_ingredient
      Loaders::RecordLoader
        .for(RecipeIngredient, column_name: :ingredient_id, scope: object.parent.recipe_ingredients)
        .load(object.node.id)
    end
  end

  class RecipeIngredientConnectionType < Types::BaseConnection
    edge_type Types::RecipeIngredientEdgeType
  end

  class RecipeType < Types::BaseObject
    field :id, ID, null: false
    field :name, String, null: false
    field :description, String, null: true
    field :created_at, GraphQL::Types::ISO8601DateTime, null: false

    field :ingredients, Types::RecipeIngredientConnectionType, null: false
  end
end

module Loaders
  class RecordLoader < GraphQL::Batch::Loader
    # @param model [Class<ApplicationRecord>]
    # @param column_name [Symbol, String]
    # @param scope [ActiveRecord::Relation, nil]
    # @return [Array]
    def self.loader_key_for(model, column_name: model.primary_key, scope: nil)
      [self, model, column_name.to_s, scope&.to_sql]
    end

    # @param model [Class<ApplicationRecord>]
    # @param column_name [Symbol, String]
    # @param scope [ActiveRecord::Relation, nil]
    def initialize(model, column_name: model.primary_key, scope: nil)
      @model       = model
      @column_name = column_name.to_s
      @column_type = model.type_for_attribute(@column_name)
      @scope       = scope
    end

    def load(key)
      super(@column_type.cast(key))
    end

    def perform(keys)
      query(keys).each { |record| fulfill(record.public_send(@column_name), record) }
      keys.each { |key| fulfill(key, nil) unless fulfilled?(key) }
    end

  private

    def query(keys)
      scope = @model
      scope = scope.merge(@scope) if @scope

      scope.where(@column_name => keys)
    end
  end
end
```

**GraphQL query**

Example GraphQL query (no response possible because of exception)

```graphql
query {
  recipes {
    id
    name
    ingredients {
      edges {
        amount
        unit
        node {
          id
          name
        }
      }
    }
  }
}
```

**Steps to reproduce**

Steps to reproduce the behavior

**Expected behavior**

Parent object is accessible from Edge type.

**Actual behavior**

```
undefined method `parent' for #<GraphQL::Pagination::Connection::Edge:0x00007fe085b7da00>
Did you mean?  present?
```

Place full backtrace here (if a Ruby exception is involved):

<details>
<summary>Click to view exception backtrace</summary>

```
undefined method `parent' for #<GraphQL::Pagination::Connection::Edge:0x00007fe085b7da00>
Did you mean?  present?
/Users/mintyfresh/Projects/drink/app/graphql/types/recipe_ingredient_edge_type.rb:23:in `recipe_ingredient'
/Users/mintyfresh/Projects/drink/app/graphql/types/recipe_ingredient_edge_type.rb:11:in `amount'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/schema/field.rb:578:in `public_send'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/schema/field.rb:578:in `block in resolve'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/schema/field.rb:732:in `with_extensions'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/schema/field.rb:561:in `resolve'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:226:in `block (5 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/tracing.rb:67:in `trace'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:225:in `block (4 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/query.rb:354:in `block in with_error_handling'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/errors.rb:30:in `with_error_handling'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/query.rb:353:in `with_error_handling'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:224:in `block (3 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:382:in `resolve_with_directives'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:221:in `block (2 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:177:in `block in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:334:in `block in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:329:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:355:in `block (2 levels) in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:352:in `block in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:345:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:345:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:238:in `block (4 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:232:in `block (3 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:382:in `resolve_with_directives'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:221:in `block (2 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:177:in `block in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:334:in `block in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:329:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:375:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:238:in `block (4 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:232:in `block (3 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:382:in `resolve_with_directives'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:221:in `block (2 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:177:in `block in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:334:in `block in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:329:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:355:in `block (2 levels) in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:352:in `block in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:345:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:345:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:238:in `block (4 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:232:in `block (3 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:382:in `resolve_with_directives'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:221:in `block (2 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:177:in `block in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:334:in `block in continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:329:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:375:in `continue_field'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:238:in `block (4 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:232:in `block (3 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:382:in `resolve_with_directives'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:221:in `block (2 levels) in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:457:in `after_lazy'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:177:in `block in evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:124:in `evaluate_selections'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter/runtime.rb:60:in `run_eager'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter.rb:73:in `block in evaluate'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/tracing.rb:67:in `trace'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter.rb:72:in `evaluate'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/interpreter.rb:45:in `begin_query'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:113:in `begin_query'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:84:in `block in run_as_multiplex'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:83:in `map'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:83:in `run_as_multiplex'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:62:in `block (2 levels) in run_queries'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:196:in `block in instrument_and_analyze'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:29:in `block (2 levels) in apply_instrumenters'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:46:in `block (2 levels) in each_query_call_hooks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:41:in `each_query_call_hooks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:45:in `block in each_query_call_hooks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:72:in `call_hooks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:44:in `each_query_call_hooks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:27:in `block in apply_instrumenters'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:72:in `call_hooks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/instrumentation.rb:26:in `apply_instrumenters'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:175:in `instrument_and_analyze'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:61:in `block in run_queries'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/tracing.rb:67:in `trace'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:59:in `run_queries'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/execution/multiplex.rb:49:in `run_all'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/schema.rb:1633:in `multiplex'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.10/lib/graphql/schema.rb:1604:in `execute'
/Users/mintyfresh/Projects/drink/app/controllers/graphql_controller.rb:5:in `execute'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/abstract_controller/base.rb:195:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal/rendering.rb:30:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/abstract_controller/callbacks.rb:42:in `block in process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/callbacks.rb:101:in `run_callbacks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/abstract_controller/callbacks.rb:41:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal/rescue.rb:22:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal/instrumentation.rb:33:in `block in process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/notifications.rb:180:in `block in instrument'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/notifications.rb:180:in `instrument'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal/instrumentation.rb:32:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activerecord-6.0.3.1/lib/active_record/railties/controller_runtime.rb:27:in `process_action'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/abstract_controller/base.rb:136:in `process'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal.rb:190:in `dispatch'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_controller/metal.rb:254:in `dispatch'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/routing/route_set.rb:33:in `serve'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/journey/router.rb:49:in `block in serve'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/journey/router.rb:32:in `each'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/journey/router.rb:32:in `serve'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/routing/route_set.rb:834:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/rack-2.2.2/lib/rack/etag.rb:27:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/rack-2.2.2/lib/rack/conditional_get.rb:40:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/rack-2.2.2/lib/rack/head.rb:12:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activerecord-6.0.3.1/lib/active_record/migration.rb:567:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/callbacks.rb:101:in `run_callbacks'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/callbacks.rb:26:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/executor.rb:14:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/railties-6.0.3.1/lib/rails/rack/logger.rb:37:in `call_app'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/railties-6.0.3.1/lib/rails/rack/logger.rb:26:in `block in call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/tagged_logging.rb:80:in `block in tagged'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/tagged_logging.rb:28:in `tagged'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/tagged_logging.rb:80:in `tagged'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/railties-6.0.3.1/lib/rails/rack/logger.rb:26:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/request_id.rb:27:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/rack-2.2.2/lib/rack/runtime.rb:22:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/activesupport-6.0.3.1/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/executor.rb:14:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/static.rb:126:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/rack-2.2.2/lib/rack/sendfile.rb:110:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/actionpack-6.0.3.1/lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/railties-6.0.3.1/lib/rails/engine.rb:527:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/puma-4.3.5/lib/puma/configuration.rb:228:in `call'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/puma-4.3.5/lib/puma/server.rb:713:in `handle_request'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/puma-4.3.5/lib/puma/server.rb:472:in `process_client'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/puma-4.3.5/lib/puma/server.rb:328:in `block in run'
/Users/mintyfresh/.rvm/gems/ruby-2.6.5/gems/puma-4.3.5/lib/puma/thread_pool.rb:134:in `block in spawn_thread'
```

</details>

**Additional context**

Removing:
```ruby
  use GraphQL::Pagination::Connections
```
From the GraphQL schema restores necessary functionality.

**Comment by rmosolgo** [comment]
Hey, thanks for the detailed report and sorry about the incompatibility. It should be possible to fix by threading the object into `Connections#wrap`, then into the connection wrapper, then into the edge.

**Comment by rmosolgo** [comment]
Fixed by #2961, to be released in 1.11.0 (hopefully by the end of this week!). Feel free to give it a try from GitHub: 

```
gem "graphql", github: "rmosolgo/graphql-ruby", branch: "1.11-dev"
```
